src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/UseLiteralsWhereAppropriate.cs

```diff
@@ -95,5 +101,12 @@ public override void Initialize(AnalysisContext context)
                 OperationKind.FieldInitializer);
             });
         }
+
+        private static bool IsConstantInterpolatedStringSupported()
+        {
+            // TODO: When constant interpolated string is supported in a stable language version (most likely C# 10), this method should be updated.
```

Comment:
Can you instead check the LanguageVersion from the Parse options and get rid of the TODO? You would need to add analyzer sub-types to C# and VB layers to access the parse options.